frontend/src/components/PrivateRoute.js

```diff
@@ -30,13 +30,13 @@ const PrivateRoute = ({ children }) => {
       }
 
       try {
+        // note that if getUser fails with 401 or 403, axios will redirect user to login too
         const response = await getUser()
         setUser(response.data)
         setViewState(PrivateRouteViewState.authed)
       } catch (err) {
         // JWT possibly expired or invalid
         console.error('Error initializaing app', err)
-        setViewState(PrivateRouteViewState.unauthed)
```

Comment:
Note that we don't need this anymore cos axios will auto detect the 401/403 and redirect the user already